nautobot/docs/user-guide/index.md

```diff
@@ -1 +1,62 @@
-# User Guide
+# User Guide Overview
+
+Welcome to the Nautobot User Guide! The User Guide is designed to help you setup, use and operate your Nautobot installation. 
+
+There are four main headings within the guide: 
+
+- **Administration**: Go here for [installation](./administration/installation/index.md), [migration](./administration/migration/migrating-from-netbox.md), [configuration](./administration/configuration/index.md) and [upgrade instructions](./administration/upgrading/upgrading.md).
+- **Feature Guides**: Walk through these guides if you are a [new user to Nautobot](./feature-guides/getting-started/index.md) and wondering how perform common tasks in the Web UI. 
+- **Core Data Model**: Read the [data models](./core-data-model/overview/introduction.md) as a more advanced user to better understand how the core application is structured. 
+- **Platform Functionality**: Explore this area for deep-dives on specific Nautobot functionality such as [Jobs](./platform-functionality/jobs/index.md), [APIs](./platform-functionality/rest-api/overview.md) or [Git repository integration](./platform-functionality/gitrepository.md).
+
+## Quick Links
+
+!!! Tip
+    Are a **new user** looking to get started with Nautobot?
+
+<!-- pyml disable-num-lines 30 no-inline-html,proper-names -->
+<div class="grid cards" markdown>
+
+- :material-cloud-download:{ .lg .middle } **Installing Nautobot**
+
+    ---
+    Get up and running with a [Nautobot install](./administration/installation/index.md) on your own Linux Platform or in a Docker environment.
+
+
+- :material-play-network:{ .lg .middle } **Using Nautobot**
+
+    ---
+    Dive into [how to use Nautobot](./feature-guides/getting-started/index.md) and the key components of the core web interface.
+
+- :material-transition-masked:{ .lg .middle } **Migrating with Nautobot**
+
+    ---
+    Dig into how you can [migrate from Netbox](./administration/migration/migrating-from-netbox.md) to Nautobot seamlessly. 
+</div>
+
+!!! Tip
+    Are you an **existing User** with Nautobot looking to dig deeper?
+
+<!-- pyml disable-num-lines 30 no-inline-html,proper-names -->
+<div class="grid cards" markdown>
+
+- :material-api:{ .lg .middle } **Nautobot APIs**
+
+    ---
+    Check out the [REST API documentation](./platform-functionality/rest-api/overview.md) covering both basic features and advanced use cases. 
+    
+    Are you looking for [GraphQL](./platform-functionality/graphql.md) APIs instead? If you are new to GraphQL, you can also read the [Nautobot GraphQL User Guide](./feature-guides/graphql.md)
+
+- :material-briefcase:{ .lg .middle } **Nautobot Jobs**
+
+    ---
+    Looking to use Nautobot to host your custom coding logic on demand? 
+    
+    Check out [Nautobot Jobs](./platform-functionality/jobs/index.md) which can also leverage your Nautobot data to automate using your Python code. 
+
+- :material-fire:{ .lg .middle } **Nautobot & NAPALM**
```

Comment:
this is going away, so should likely consider something else.

Reply:
perhaps app development or other apps in general.

Reply:
I assume you're talking about the NAPALM part, but the selection above makes it look like it's jobs going away :) Since this is 2.0 docs, I don't see harm in leaving that panel there for now.

Good call on adding apps - but not yet development, let's point at existing apps they can install to extend the functionality of Nautobot.

Reply:
yes, only speaking about napalm, it's just the way github shows context, even though it's only earmarked to a single line.

Reply:
Sounds good, didn't know about the NAPALM going away, but will in add in an Apps one. I will point to 
https://networktocode.com/nautobot/nautobot-apps/
since that is a full list of apps more than the apps docs page
https://docs.nautobot.com/projects/core/en/stable/apps/